python/doc/usecases/use_case_branin.rst

```diff
@@ -68,20 +68,20 @@ Here we assume a constant noise variance:
 References
 ----------
 
-    * Dixon, L. C. W., & Szego, G. P. (1978). The global optimization problem: an introduction. Towards global optimization, 2, 1-15.
+* Dixon, L. C. W., & Szego, G. P. (1978). The global optimization problem: an introduction. Towards global optimization, 2, 1-15.
 
-    * Forrester, A., Sobester, A., & Keane, A. (2008). Engineering design via surrogate modelling: a practical guide. Wiley.
+* Forrester, A., Sobester, A., & Keane, A. (2008). Engineering design via surrogate modelling: a practical guide. Wiley.
 
-    * Global Optimization Test Problems. Retrieved June 2013, from http://www-optima.amp.i.kyoto-u.ac.jp/member/student/hedar/Hedar_files/TestGO.htm.
+* Global Optimization Test Problems. Retrieved June 2013, from http://www-optima.amp.i.kyoto-u.ac.jp/member/student/hedar/Hedar_files/TestGO.htm.
 
-    * Molga, M., & Smutnicki, C. Test functions for optimization needs (2005). Retrieved June 2013, from http://www.zsd.ict.pwr.wroc.pl/files/docs/functions.pdf.
+* Molga, M., & Smutnicki, C. Test functions for optimization needs (2005). Retrieved June 2013, from http://www.zsd.ict.pwr.wroc.pl/files/docs/functions.pdf.
 
-    * Picheny, V., Wagner, T., & Ginsbourger, D. (2012). A benchmark of kriging-based infill criteria for noisy optimization.
+* Picheny, V., Wagner, T., & Ginsbourger, D. (2012). A benchmark of kriging-based infill criteria for noisy optimization.
```

Comment:
Are these references not in the bibliography?

Reply:
Not at the present time. Do you suggest that I create the corresponding items in the bibliography, and use them in the branin help page?

Reply:
That would be ideal, but if you don't have time it is not that important.

Reply:
I suggest to fix that in a separate PR, because there are many pages like this, e.g.
- https://openturns.github.io/openturns/latest/usecases/use_case_flood_model.html
- https://openturns.github.io/openturns/latest/usecases/use_case_ishigami.html
- https://openturns.github.io/openturns/latest/usecases/use_case_logistic.html
- https://openturns.github.io/openturns/latest/usecases/use_case_ackley.html
- https://openturns.github.io/openturns/latest/usecases/use_case_chaboche.html
etc.
This is why I tag this topic as resolved.